### PR TITLE
Update papers

### PR DESCRIPTION
**Pull request by lukasturcani**
Related Issues: #286 

Updates the publication list.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -60,39 +60,76 @@ Publications
...
 
-.. _`stk: A Python Toolkit for Supramolecular Assembly`: https://onlinelibrary.wiley.com/doi/abs/10.1002/jcc.25377
+__ https://onlinelibrary.wiley.com/doi/abs/10.1002/jcc.25377
+__ https://chemrxiv.org/articles/STK_A_Python_Toolkit_for_Supramolecular_Assembly/6127826
 
 using stk
 ---------
 
-* `An Evolutionary Algorithm for the Discovery of Porous Organic Cages`_ | chemrxiv__
+* `Forecasting System of Computational Time of DFT/TDDFT Calculations
```
Comment (andrewtarzia) [inline README.rst:77]: Missing my paper on chemrxiv: https://chemrxiv.org/articles/preprint/High-throughput_Computational_Evaluation_of_Low_Symmetry_Pd2L4_Cages_to_Aid_in_System_Design/14604294

File: docs/source/video_tutorials.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
+Video Tutorials
```
Comment (andrewtarzia) [inline docs/source/video_tutorials.rst:1]: Can I have my name somewhere on this page?

File: docs/source/video_tutorials.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+===============
+
+Basics of *stk*
+---------------
+
+https://www.youtube.com/watch?v=mPr9D7nCQ84&list=PLIWYdPQ9hLzVngMF8NOkiApMtgc_ZwZgO&index=1
```
Comment (andrewtarzia) [inline docs/source/video_tutorials.rst:7]: Can you use something like below for links?

```
[Video 1](https://www.youtube.com/watch?v=mPr9D7nCQ84&list=PLIWYdPQ9hLzVngMF8NOkiApMtgc_ZwZgO&index=1)
```

File: docs/source/video_tutorials.rst [hunk 4]
```diff
@@ -0,0 +1,39 @@
+Video Tutorials
+===============
+
+Basics of *stk*
+---------------
```
Comment (andrewtarzia) [inline docs/source/video_tutorials.rst:5]: Overview of the recent publication covering the capability of _stk_ and the docs of _stk_. 

Jupyter Notebook is available at https://github.com/andrewtarzia/stk-examples/tree/main/notebooks_from_videos .

File: docs/source/video_tutorials.rst [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+https://www.youtube.com/watch?v=mPr9D7nCQ84&list=PLIWYdPQ9hLzVngMF8NOkiApMtgc_ZwZgO&index=1
+
+Functional Groups in *stk*
+--------------------------
```
Comment (andrewtarzia) [inline docs/source/video_tutorials.rst:10]: Overview of defining functional groups in _stk_. Includes an example of the versatility of _stk_ to unique problems.

Jupyter Notebook is available at https://github.com/andrewtarzia/stk-examples/tree/main/notebooks_from_videos .

File: docs/source/video_tutorials.rst [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+https://www.youtube.com/watch?v=IaLIN7jXyKQ&list=PLIWYdPQ9hLzVngMF8NOkiApMtgc_ZwZgO&index=2
+
+Cage Building
+-------------
```
Comment (andrewtarzia) [inline docs/source/video_tutorials.rst:15]: Overview of organic and metal-organic cage construction in _stk_. This example covers how _stk_ handles isomerism and the concept of hierarchical construction.

Jupyter Notebook is available at https://github.com/andrewtarzia/stk-examples/tree/main/notebooks_from_videos .

File: docs/source/video_tutorials.rst [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+https://www.youtube.com/watch?v=sc1Kf6vzTSA&list=PLIWYdPQ9hLzVngMF8NOkiApMtgc_ZwZgO&index=3
+
+
+Hierarchical Construction with *stk*
+------------------------------------
```
Comment (andrewtarzia) [inline docs/source/video_tutorials.rst:21]: This video expands on the use hierarchical construction in _stk_. This includes an implementation of the example from the most recent publication and the construction a rotaxane/cage hybrid!

Jupyter Notebook is available at https://github.com/andrewtarzia/stk-examples/tree/main/notebooks_from_videos .

File: docs/source/video_tutorials.rst [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+https://www.youtube.com/watch?v=vvmS43dVPe4&list=PLIWYdPQ9hLzVngMF8NOkiApMtgc_ZwZgO&index=5
+
+
+Databases and *stk-vis* in *stk*
+--------------------------------
+
+https://www.youtube.com/watch?v=ESfmmLpHVmE&list=PLIWYdPQ9hLzVngMF8NOkiApMtgc_ZwZgO&index=6
+
+Host-Guest Conformers with *stk*
+--------------------------------
```
Comment (andrewtarzia) [inline docs/source/video_tutorials.rst:37]: An example of the generation of _stk_ host-guest complexes using the software SpinDry (https://github.com/andrewtarzia/SpinDry)

Jupyter Notebook is available at https://github.com/andrewtarzia/stk-examples/tree/main/notebooks_from_videos .

File: docs/source/video_tutorials.rst [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+https://www.youtube.com/watch?v=WB9oAqA8m4o&list=PLIWYdPQ9hLzVngMF8NOkiApMtgc_ZwZgO&index=4
+
+Using *stko* with *stk*
+-----------------------
+
+https://www.youtube.com/watch?v=vvmS43dVPe4&list=PLIWYdPQ9hLzVngMF8NOkiApMtgc_ZwZgO&index=5
+
+
+Databases and *stk-vis* in *stk*
+--------------------------------
```
Comment (andrewtarzia) [inline docs/source/video_tutorials.rst:32]: A short introduction to databasing and the use of _stk-vis_ with _stk_ molecules!

Jupyter Notebook is available at https://github.com/andrewtarzia/stk-examples/tree/main/notebooks_from_videos .

File: docs/source/video_tutorials.rst [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+https://www.youtube.com/watch?v=sc1Kf6vzTSA&list=PLIWYdPQ9hLzVngMF8NOkiApMtgc_ZwZgO&index=3
+
+
+Hierarchical Construction with *stk*
+------------------------------------
+
+https://www.youtube.com/watch?v=WB9oAqA8m4o&list=PLIWYdPQ9hLzVngMF8NOkiApMtgc_ZwZgO&index=4
+
+Using *stko* with *stk*
+-----------------------
```
Comment (andrewtarzia) [inline docs/source/video_tutorials.rst:26]: This video shows the usage of the _stko_ package to optimise and calculate the properties of _stk_ molecules.

Jupyter Notebook is available at https://github.com/andrewtarzia/stk-examples/tree/main/notebooks_from_videos .

**Review by andrewtarzia — CHANGES_REQUESTED** [review]

File: docs/source/video_tutorials.rst (hunk 2 above)
Comment (lukasturcani) [reply docs/source/video_tutorials.rst:1]: For sure, but I was hoping you would add it. But I guess something like "Below is a list of video tutorials made by Andrew Tarzia, which provide a thorough introduction to stk."

Comment (andrewtarzia) [reply docs/source/video_tutorials.rst:1]: Perfect! Link to my github with my name?

Comment (lukasturcani) [reply docs/source/video_tutorials.rst:1]: :+1:  Sounds good to me

**Review by andrewtarzia — APPROVED** [review]

**Pull request merged**